inc/classes/class-astra-home-page-banner-markup.php

```diff
@@ -71,43 +71,43 @@ function body_classes( $classes ) {
 		 *
 		 * @since 1.0.0
 		 */
-		function load_markup( ) {
+		public function load_markup() {
 
 			if ( ! defined( 'ASTRA_THEME_SETTINGS' ) ) {
 				return;
 			}
 
-			if( is_front_page() ) {
-				$banner_heading 	= astra_get_option('ast-banner-heading');
-				$banner_subheading 	= astra_get_option('ast-banner-subheading');
-				$home_page_banner_size 	= astra_get_option('banner-image-size-option');
+			if ( is_front_page() ) {
+				$banner_heading        = astra_get_option( 'ast-banner-heading' );
+				$banner_subheading     = astra_get_option( 'ast-banner-subheading' );
+				$home_page_banner_size = astra_get_option( 'banner-image-size-option' );
 
 				// Full Screen.
 				$full_screen = ( 'full-size' == $home_page_banner_size ) ? ' ast-full-home-page-banner' : '';
 
-				$html = '<div class="home-page-banner'.$full_screen.'">' .
+				$html = '<div class="home-page-banner' . $full_screen . '">' .
 							'<div class="heading-container">' .
-								'<h2 class="banner-heading">'.$banner_heading.'</h2>' .
-								'<h3 class="banner-subheading">'.$banner_subheading.'</h3>' .
+								'<h2 class="banner-heading">' . $banner_heading . '</h2>' .
+								'<h3 class="banner-subheading">' . $banner_subheading . '</h3>' .
 							'</div>' .
 						'</div>';
-				echo $html;
+				echo wp_kses_post( $html );
```

Comment:
@shubhw12 Please remove the `wp_kses_post` from here.

Reply:
Done